include/wx/gtk/private/mediactrl.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _WX_GTK_PRIVATE_MEDIACTRL_H_
+#define _WX_GTK_PRIVATE_MEDIACTRL_H_
+
+#ifdef GDK_WINDOWING_X11
+    #include <gdk/gdkx.h>
+#endif
+#ifdef GDK_WINDOWING_WAYLAND
+    #include <gdk/gdkwayland.h>
+#endif
+
+//-----------------------------------------------------------------------------
+// "wxGtkGetIdFromWidget" from widget
+//
+// Get the windows_id performing run-time checks If the window wasn't realized when Load was called, this is the
+// callback for when it is - the purpose of which is to tell
+// GStreamer to play the video in our control
```

Comment:
Very trivial, but could you please wrap lines to ~80 length, i.e.
```suggestion
// Get the windows_id performing run-time checks If the window wasn't realized
// when Load was called, this is the callback for when it is - the purpose of
// which is to tell GStreamer to play the video in our control.
```

Reply:
done